events.lua

```diff
@@ -1,6 +1,14 @@
 function register_events(event)
 	gauge_tick:set(game.tick)
 
+        for _, surface in pairs(game.surfaces) do
+                gauge_seed:set(surface.map_gen_settings.seed, {surface.name})
+        end
+
+        for name, version in pairs(game.active_mods) do
+                gauge_mods:set(1, {name, version})
```

Comment:
i think mod list and version can't change during a live game, so it's best to just fill this value at save loading instead of query it for each n tick